data/fixtures/scopes/python/argument.actual2.scope

```diff
@@ -0,0 +1,10 @@
+" ".join(word for word in word_list)
+---
+
+[Content] =
+[Removal] =
+[Domain] = 0:9-0:35
+           >--------------------------<
+0| " ".join(word for word in word_list)
```

Comment:
Personally I don't really get this implementation. This appears to be how it was implemented in legacy and all the tests passed. What is the purpose of this scope?